optlib/terraform.ctags

```diff
@@ -16,13 +16,20 @@
 # it's way harder to write a good regex for that.
 #   - https://www.terraform.io/docs/language/values/locals.html
 
---langdef=tf
---map-tf=+tf:.tf.tfvars
---regex-tf=/^[[:space:]]*resource[[:space:]]*"([^"]*)"[[:space:]]*"([^"]*)"/\2/r,Resource/
---regex-tf=/^[[:space:]]*data[[:space:]]*"([^"]*)"[[:space:]]*"([^"]*)"/\2/d,Data/
---regex-tf=/^[[:space:]]*variable[[:space:]]*"([^"]*)"/\1/v,Variable/
---regex-tf=/^[[:space:]]*provider[[:space:]]*"([^"]*)"/\1/p,Provider/
---regex-tf=/^[[:space:]]*module[[:space:]]*"([^"]*)"/\1/m,Module/
---regex-tf=/^[[:space:]]*output[[:space:]]*"([^"]*)"/\1/o,Output/
---regex-tf=/^([a-z0-9_]+)[[:space:]]*=/\1/f,TFVar/
+--langdef=terraform
+--map-terraform=+.tf
+--map-terraform=+.tfvars
+--kinddef-terraform=r,Resource,Terraform Resource
+--kinddef-terraform=d,Data,Terraform Data
+--kinddef-terraform=v,Variable,Terraform Variable
+--kinddef-terraform=p,Provider,Terraform Provider
+--kinddef-terraform=m,Module,Terraform Module
+--kinddef-terraform=o,Output,Terraform Output
+--regex-terraform=/^[[:space:]]*resource[[:space:]]*"([^"]*)"[[:space:]]*"([^"]*)"/\2/r,Resource/
+--regex-terraform=/^[[:space:]]*data[[:space:]]*"([^"]*)"[[:space:]]*"([^"]*)"/\2/d,Data/
+--regex-terraform=/^[[:space:]]*variable[[:space:]]*"([^"]*)"/\1/v,Variable/
```

Comment:
v is already defined with `--kinddef-...`. So you don't have to specify `v,Variable`. Just `v` is enough:
```
--regex-Terraform=/^[[:space:]]*variable[[:space:]]*"([^"]*)"/\1/v/
```

I don't know well about the syntax of Terraforms. However, I guess a whitespace may be needed after `variable` keyword. We don't record a variable having an empty string as name. So `+` is better than `*`.

```
--regex-Terraform=/^[[:space:]]*variable[[:space:]]+"([^"]+)"/\1/v/
```

Reply:
This is applicable to other lines of `--regex-...`.